### ISSUE TITLE
Can't compile frozen facenet graph (Proper version)

### ISSUE THREAD

**Issue by Dima00782** [tensorflow/tensorflow#21728]
### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: NO
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Linux Ubuntu 16.04 
- **Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device**: not mobile
- **TensorFlow installed from (source or binary)**: source
- **TensorFlow version (use command below)**: 1.9
- **Python version**: 2.7
- **Bazel version (if compiling from source)**: 0.16.1
- **GCC/Compiler version (if compiling from source)**: clang
- **CUDA/cuDNN version**: no
- **GPU model and memory**: no
- **Exact command to reproduce**: `./tfcompile --graph=frozen_20170512-110547.pb --config=frozen_20170512-110547.pbtxt --cpp_class="my_super_class" --target_features="+avx2"`

### Describe the problem
I've tried to compile facenet https://github.com/davidsandberg/facenet frozen graph - 20180408-102900
And I've got this error:
`INVALID ARGUMENTS: Unable to functionalize control flow in graph: Switch ('InceptionResnetV1/Conv2d_1a_3x3/BatchNorm/cond/Switch_1') has operands ('InceptionResnetV1/Conv2d_1a_3x3/BatchNorm/cond/Switch_1/Switch' and 'InceptionResnetV1/Conv2d_1a_3x3/BatchNorm/cond/pred_id') that have different switch depths (1 != 0)
`

Is there any workaround for this? Does Tensorflow support compiling BatchNorm?
### Source code / logs
My .pbtxt:
```
feed {
  id { node_name: "input" }
  shape {
    dim { size: 160 }
    dim { size: 160 }
  }
}

fetch {
  id { node_name: "embeddings" }
}
```

**Comment by tensorflowbutler** [comment]
Nagging Assignee @tatatodd: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @tatatodd: It has been 29 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @tatatodd: It has been 44 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @tatatodd: It has been 74 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @tatatodd: It has been 91 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @tatatodd: It has been 105 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by ymodak** [comment]
Apologies for delay in response. I think facenet repo can be better platform to address this question, since its not a bug or feature request on the lines of TensorFlow core framework. You can post this issue in facenet repo from [here](https://github.com/davidsandberg/facenet/issues). Thanks!
